ivtest/regress-sv.list

```diff
@@ -237,6 +237,7 @@ enum_in_struct		normal,-g2005-sv	ivltests
 enum_in_class		normal,-g2005-sv	ivltests
 enum_in_class_name_coll	CE,-g2005-sv		ivltests
 enum_next		normal,-g2005-sv	ivltests
+enum_order		CO,-g2005-sv		ivltests
```

Comment:
Can you make this a "normal" program, even if you just add:
```
   initial $display("PASSED");
```
Yeah, that doesn't do much, but it does assure that the code generator doesn't choke on the change.

Reply:
Added a PASSED that checks if all enum identifiers have the same value.